test/platform-stats.test.js

```diff
@@ -0,0 +1,62 @@
+import assert from 'node:assert'
+import pg from 'pg'
+import { beforeEach, describe, it } from 'mocha'
+
+import { DATABASE_URL } from '../lib/config.js'
+import { migrateWithPgClient } from '../lib/migrate.js'
+import { VALID_MEASUREMENT } from './helpers/test-data.js'
+import { updateDailyNodeMetrics } from '../lib/platform-stats.js'
+
+const createPgClient = async () => {
+  const pgClient = new pg.Client({ connectionString: DATABASE_URL })
+  await pgClient.connect()
+  return pgClient
+}
+
+describe('platform-stats', () => {
+  let pgClient
+
+  before(async () => {
+    pgClient = await createPgClient()
+    await migrateWithPgClient(pgClient)
+  })
+
+  beforeEach(async () => {
+    await pgClient.query('DELETE FROM daily_node_metrics')
+    await pgClient.query('BEGIN TRANSACTION')
+  })
+
+  afterEach(async () => {
+    await pgClient.query('END TRANSACTION')
+  })
+
+  after(async () => {
+    await pgClient.end()
+  })
+
+  it('updates daily node metrics with new measurements', async () => {
```

Comment:
Since we are anticipating adding more platform stats very soon, let's avoid code churn and nest this new test under `describe('updateDailyNodeMetrics')` (or a similarly named) test group.

```js
describe('updateDailyNodeMetrics', () => {
  it('updates daily node metrics with new measurements', async () => { 
    //...
  })

  it('ignores duplicate measurements for the same station on the same day',
  // ...
  )
})
```